backend/fms_core/models/metric.py

```diff
@@ -23,6 +23,12 @@ class Metric(TrackedModel):
     value_numeric = models.DecimalField(null=True, blank=True, max_digits=METRIC_PRECISION, decimal_places=METRIC_DECIMAL_PLACES, help_text="Metric numerical value.")
     value_string = models.CharField(null=True, blank=True, max_length=STANDARD_STRING_FIELD_LENGTH, help_text="Metric string value.")
 
+    class Meta:
+        indexes = [
+            models.Index(fields=['name'], name='metric_name_idx'),
+            models.Index(fields=['name', 'value_numeric'], name='metric_name_value_numeric_idx'),
+        ]
```

Comment:
Why index on numeric value in particular?

Reply:
The reads and bases are pretty much the only metrics so far we use and i thought we might want to filter by these at some point. Get readsets for reads > 1000000 ...